Watchman.Discord/Program.cs

```diff
@@ -21,6 +22,8 @@ public static async Task Main(string[] args)
             
             var watchman = new WatchmanBot(configuration);
             var workflowBuilder = watchman.GetWorkflowBuilder();
+            workflowBuilder.ReactionAdded += (userMessage, socketMessageChannel, socketReaction) => ReactionsService.OnReactionAddedEvent(socketReaction);
```

Comment:
to wygląda jak krzywy hack 

czasami trzeba takie robić bo nie ma wyjścia
ale tutaj wygląda to jak droga na skróty i dodatkowe komplikowanie kodu 

wyobraź sobie co by było, gdyby w tym miejscu wszystkie zależności byłyby tak ogarnięte